software/programming_instructions.md

```diff
@@ -57,35 +57,24 @@ To start with, you'll need to download the [Thonny IDE](https://thonny.org/). Th
 
 ### Calibration
 
-The europi.py program is made to be imported as a library, but you are also able to run it directly to calibrate the module.  
-  
-If you wish to calibrate the module yourself, you will need access to two fixed voltage sources, ideally 1V and 10V.  
-  
-**NOTE: For calibration, the module *must* be connected to both USB for programming *and* rack power for ±12V.**
-  
-As soon as you run the program, a 'calibration.txt' file will be generated, which will allow you to use the module, but they are only default values and will not allow high accuracy input or output.  
-  
-#### Calibration.txt
-| Line | Meaning |
-| ------------- | ----------- |
-|1|ADC voltage multiplier
-|2|ADC offset (in volts)
-|3|Output voltage multiplier
-  
-If you don't need high accuracy, you can finish at this point and begin programming (skip to the next section).
-If however you'd like your module to be able to accurately read and output specific voltages, then get ready your voltage source(s).  
-A benchtop power supply is ideal for this, but you could potentially use another Eurorack module if the voltage is accurate enough (it's reccommended to use a multimeter to make sure the voltage you input to the program variables is accurate).  
-
-1. If the voltages you plan to use are any different to 1V and 10V, open the europi.py file and change the values of LOW_VOLTAGE and HIGH_VOLTAGE to whichever you are using. As long as they are between 0-10V and are far enough apart from each other, the process will work.
+To use the module for accurately reading and outputting voltages, you need to complete a calibration process. This will allow your specific module to account for any differences in components, such as resistor tolerances.  
+If you do not wish to calibrate the module and don't mind your voltages being slightly inaccurate, simply skip to the programming step and your module will use default values.
+
+1. To begin, you need to copy the [calibrate.py](/software/firmware/calibrate.py) file from the firmware folder to your Pico using the same process as you did for europi.py.
+2. Save the file into the lib folder, next to europi.py, but instead of naming it 'calibrate.py', name it 'main.py'. This is so that the module knows to run this program automatically when connected to power, without you needing to have USB connected. This is also the way you choose what program will run when you power on the module later on.
```

Comment:
Optional nit: Alternatively, you can save the file name as `calibrate.py` along side the `main.py` file and use Thonny to run the file instead of naming/renaming calibrate with `main.py`. This would simplify step 10 a bit too.